text/0092-logs-vision.md

```diff
@@ -96,6 +96,16 @@ This is important because we believe software that was created before
 OpenTelemetry should not be disregarded and should benefit from OpenTelemetry
 efforts where possible.
 
+Specifically, we should be able to collect the following sources:
+- Log files, especially those emitted by container runtimes
+- Syslog
+- Systemd/Journald
+- Windows Event Log
```

Comment:
I think these four are obvious and won't lead to any disagreement, the others were added for completeness, but I'm happy to remove them if thats desired

Reply:
I agree, these look like basic things that we do want to support. We may need to discuss and decide the priorities.

These are requirements for the Collector, right? We do not intend to collect these using e.g. OpenTelemetry SDK. If these are for the Collector then best to make these Collector roadmap items.